internal-packages/next-dev/src/shared.ts

```diff
@@ -21,56 +35,64 @@ export function monkeyPatchVmModule({ env, cf, ctx, caches }: PlatformProxy) {
 
 	const originalRunInContext = vmModule.runInContext.bind(vmModule);
 
-	vmModule.runInContext = (
-		...args: [
-			string,
-			Record<string, unknown> & {
-				process?: { env?: Record<string, unknown> };
-				[cloudflareRequestContextSymbol]?: {
-					env: unknown;
-					ctx: unknown;
-					cf: unknown;
-				};
-			},
-			...[unknown],
-		]
-	) => {
+	vmModule.runInContext = (...args: [string, RuntimeContext, ...[unknown]]) => {
```

Comment:
Isn't `...[unknown]` the same thing as `unknown`?
So you could just do

```suggestion
	vmModule.runInContext = (...args: [string, RuntimeContext, unknown]) => {
```

Reply:
https://www.typescriptlang.org/play?#code/C4TwDgpgBAZgdgRigXigCgHRYIYCcDmAzgFxQDahwuAlnPgDRRwCuAtgEYS6PNwDWcAPYB3OAF0AlCgB8UAG6DqAEwDcAKFCRYcAEwp0WDHiKkKVWgyZtO3KIbK8BI8ZJnzFqtWoA2EYLARSeCRUNClkWQBvAF91X38YHSDdfRgEdSA

Reply:
Or did you really mean:

```suggestion
	vmModule.runInContext = (...args: [string, RuntimeContext, ...unknown[]]) => {
```